packages/core/src/hub.ts

```diff
@@ -233,7 +233,7 @@ export class Hub implements HubInterface {
    */
   public captureEvent(event: Event, hint?: EventHint): string {
     const eventId = hint && hint.event_id ? hint.event_id : uuid4();
-    if (event.type !== 'transaction') {
+    if (!event.type) {
```

Comment:
Question: I guess replays should not count here? (Eventually - currently they do not go through this method at all)

Reply:
We might also add a test for the new event type here: 
https://github.com/getsentry/sentry-javascript/blob/c68d429ddcd7ca772dcfb181112d73048455a5c6/packages/hub/test/hub.test.ts#L360-L373

Reply:
done in https://github.com/getsentry/sentry-javascript/pull/6481/commits/37535a05a7db85b66b8c8aac8c73db52dac74523